modules/nodes-nodes-kernel-arguments.adoc

```diff
@@ -20,7 +20,31 @@ Examples of kernel arguments you could set include:
 
 * **nosmt**: Disables symmetric multithreading (SMT) in the kernel. Multithreading allows multiple logical threads for each CPU. You could consider `nosmt` in multi-tenant environments to reduce risks from potential cross-thread attacks. By disabling SMT, you essentially choose security over performance.
 
+ifndef::openshift-origin[]
 * **systemd.unified_cgroup_hierarchy**: Enables link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2). cgroup v2 is the next version of the kernel link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01[control group] and offers multiple improvements.
++
+[IMPORTANT]
+====
+[subs="attributes+"]
+{product-title} cgroups version 2 support is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
```

Comment:
@GroceryBoyJr I took that text from the Tech Preview snippet. I wonder if we should change the text in the snippet also? I wonder what damage that would cause!

Reply:
I worried about that myself. There's been some general discussion about that in peer review (not as pertains to this PR, but as pertains to support language in general.)

Your call, sir. If you got it from the snippet, then it's been reviewed already.